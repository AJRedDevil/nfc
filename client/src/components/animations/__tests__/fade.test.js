// npm packages
import React from 'react';
import renderer from 'react-test-renderer';

// our packages
import Fade from '../fade';

test('# Fade', () => {
  const test = {message: 'test'};
  const wrapper = renderer
    .create(
      <Fade test={test}>
        <div>Test</div>
      </Fade>
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
