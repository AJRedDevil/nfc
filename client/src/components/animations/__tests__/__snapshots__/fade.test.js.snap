// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Fade 1`] = `
<div
  test={
    Object {
      "message": "test",
    }
  }
>
  Test
</div>
`;
